src/components/Sum.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
const Sum = ({ a, b }) => (
```

Reply:
these new lines for props are also extra, next time no need to put them all of them on new lines

Reply:
Okay...